plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -1677,6 +1926,11 @@
             platformFilter: countlyPushNotification.service.PlatformEnum.ALL,
             totalPushMessagesSent: 0,
             totalUserActionsPerformed: 0,
+            userCommand: {
```

Comment:
userCommand state with isDrawerOpen will be moved in its own module, so that actions, mutations in reference to the drawer will be re-used for main and details pages.